js/puzzlescript-embed.js

```diff
@@ -9,25 +9,26 @@
 			{
 				engine: "http://www.puzzlescript.net/js/scripts_play_compiled.js"
 			};
-		
+
 		extend(settings, config);
 		clear_children(element);
 		canvas = create_game_canvas(element, settings);
-		load_engine(element, id, settings, function()
+
+        return load_engine(element, id, settings, function()

```

Comment:
ew I appologize for this. My editor added spaces instead of tabs and I totally didn't notice. I can fix it!